user-storage-simple/src/main/java/org/keycloak/quickstart/writeable/PropertyFileUserStorageProvider.java

```diff
@@ -284,16 +181,96 @@ public void disableCredentialType(RealmModel realm, UserModel user, String crede
     private static final Set<String> disableableTypes = new HashSet<>();
 
     static {
-        disableableTypes.add(CredentialModel.PASSWORD);
+        disableableTypes.add(PasswordCredentialModel.TYPE);
     }
-
+    
     @Override
-    public Set<String> getDisableableCredentialTypes(RealmModel realm, UserModel user) {
+    public void close() {
 
-        return disableableTypes;
     }
+    
     @Override
-    public void close() {
-
+	public Stream<UserModel> searchForUserStream(RealmModel realm, String search) {
+    	return searchForUserStream(realm, search, 0, Integer.MAX_VALUE);
     }
+
+	@Override
+	public Stream<UserModel> searchForUserStream(RealmModel realm, String search, Integer firstResult, Integer maxResults) {
+		
+		List<UserModel> users = new LinkedList<>();
+		
+        int i = 0;
+        
+        for (Object obj : properties.keySet()) {
+            String username = (String)obj;
+            if (!username.contains(search)) continue;
+            if (i++ < firstResult) continue;
+            UserModel user = getUserByUsername(realm, username);
+            users.add(user);
+            if (users.size() >= maxResults) break;
+        }
+        return users.stream();
+	}
+
+	@Override
+	public Stream<UserModel> searchForUserStream(RealmModel realm, Map<String, String> params) {
+        return searchForUserStream(realm, params, 0, Integer.MAX_VALUE);
+	}
```

Comment:
This method has default implementation, this seems as redundant:
 
https://github.com/keycloak/keycloak/blob/9052ec2b02f0aace57c5ffb6fb472345a56b11cf/server-spi/src/main/java/org/keycloak/storage/user/UserQueryProvider.java#L228-L230